src/test/integration/api_request.py

```diff
@@ -0,0 +1,7 @@
+import json, requests
+from io import BytesIO
```

Comment:
I'll be removing this import. It was left over from the images-api integration tests.